index.js

```diff
@@ -26,6 +26,9 @@ module.exports = (input, opts) => {
 	input = input || 'patch';
 	opts = opts || {};
 
+	const runTests = !opts.yolo;
+	const runCleanup = !opts.skipCleanup && !opts.yolo;
```

Comment:
Or

	opts = Object.assign({
		tests: !opts.yolo,
		cleanup: !opts.skipCleanup && !opts.yolo
	}, opts || {});

but this now adds an undocumented `--no-cleanup` and `--no-tests` CLI options.  Which you don't want.